common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -26,7 +26,7 @@
             var match;
 
             this.$el = options.el;
-            this.readOnly = options.readOnly;
```

Comment:
If we explicitly pass `readOnly: data-read-only` in [new DiscussionInlineView({ el: $(element).find('.discussion-module') })](https://github.com/edx/edx-platform/blob/master/lms/templates/discussion/_discussion_inline.html#L29) we don't have to change this.
As we are passing whole element (el) , so I found it better to take `data-read-only` value from that element.
Please let me know if you think the otherwise method is more appropriate.
Thanks.